Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
+    minitest (5.14.4)
```

Comment:
### [minitest](https://github.com/seattlerb/minitest)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.14.3 → 5.14.4

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/seattlerb/minitest/compare/v5.14.3...v5.14.4).

These are the individual commits:
- (94faf7b) [Added a support matrix to the FAQ in README.rdoc](https://github.com/seattlerb/minitest/commit/94faf7b90cbe03b49d34d43f17cc0ea06eb8c53f)
- (9eb312b) [- Fixed deprecation warning using stub with methods using ke…](https://github.com/seattlerb/minitest/commit/9eb312bea1276117fbacb09a1c8bb7abd1434397)
- (dfbde40) [- ! Unfixed deprecation warning fix that broke in nearly eve…](https://github.com/seattlerb/minitest/commit/dfbde407800d3a5dbb54ec9461635f062c2fe272)
- (8a7ba81) [- Re-fixed deprecation warning using stub with methods using…](https://github.com/seattlerb/minitest/commit/8a7ba8191f6d09d5e7dc04d53054e8b194e61fe4)
- (13c48a0) [Improved test_stub_callable_keyword_args to test pass-throug…](https://github.com/seattlerb/minitest/commit/13c48a03d84a2a87855a4de0c959f96800100357)
- (3c6576a) [prepped for release](https://github.com/seattlerb/minitest/commit/3c6576a51f4e266996e3459d7a0dd054eb4c87f7)


</details>